src/libs/elektra/kdb.c

```diff
@@ -1222,20 +1242,24 @@ int kdbSet (KDB * handle, KeySet * ks, Key * parentKey)
 		copyError (parentKey, oldError);
```

Comment:
We have `presetstorage` which is done before (after the sanity checks like handle not NULL).

Other than that, I don't know of a hook position which suits that place.